### ISSUE TITLE
Calculate max allowed CCD temp for current catalog

### ISSUE THREAD

**Issue by jeanconn** [sot/starcheck#112]
To be able to pass back a "dynamic CCD limit", determine the max allowed CCD temperature that will satisfy requirements for acquisition and guide.  Requirements TBD.

**Comment by taldcroft** [comment]
Here is a proposal that I discussed with Sabina:

T_limit = (10.3 - M_faint) / 0.15 mag/C - 14 C

M_faint is the magnitude of the (4th / 5th) faintest star for an OR / ER, respectively.  This assumes that currently with a -14 C limit that we define a good star as anything brighter than 10.3 mag.  From the temperature perspective I think this is reasonable.

The scaling of 0.15 mag/C might need tweaking to be a bit more conservative.  (Also 0.15 is the number I recall, but I'm not 100% that is the one).

**Comment by jeanconn** [comment]
Ah.  I thought we wanted to use starcheck to be able to exclude stars from this check (so if a star already had a non-magnitude based warning it would be excluded from the limit determination).  If we don't care about the other information/warnings from starcheck, it seems to me this can all be done in the Matlab tools directly.

**Comment by taldcroft** [comment]
I don't think we were planning to have starcheck/perl be interfaced with Matlab tools.  At least I never seriously thought about it because starcheck has such a heavy dependency on the full load products and can't run standalone on an observation.

**Comment by jeanconn** [comment]
OK.  I heard you say that *this* was implemented in starcheck so I thought you actually wanted it to be there.  Granted, it isn't like the thermal model can run standalone on an observation.

**Comment by taldcroft** [comment]
When I said that *this* was implemented, I was referring to changed mag limits, so was glossing over the precise details.

Note that we still might want to compute the effective temperature limit in starcheck with better knowledge of stars being excluded for other reasons (including being a known bad star for acq/guide).  This continues on the theme of "starcheck has similar rules to SAUSAGE/ORviewer but with more details".
